infra/gcb/build_project.py

```diff
@@ -104,21 +117,26 @@ def workdir_from_dockerfile(dockerfile):
   return None
 
 
-def get_build_steps(project_dir):
-  project_yaml = load_project_yaml(project_dir)
-  dockerfile_path = os.path.join(project_dir, 'Dockerfile')
+def load_project_yaml(project_name, project_yaml, image_project):
+  """Loads project yaml and sets default values."""
+  set_yaml_defaults(project_name, project_yaml, image_project)
```

Comment:
This function seems to not be very useful? It should be unchanged from the original in that it reads the files *and* sets the defaults. It can call `set_yaml_defaults` for the setting part but the opening is missing.